### PR TITLE
Added trace_args_with_method_call action

### PR DESCRIPTION
**Pull request by gaborgsomogyi**
It's tested in the following way.

Application to trace: https://github.com/gaborgsomogyi/hadoop-token-trace

```
[gaborsomogyi:~/trace-agent] hadoop_token* 4s ± cat src/main/resources/actions.txt 
trace_args_with_method_call org.apache.hadoop.fs.FileSystem addDelegationTokens param_index:1,method_to_call:getAllTokens
...
java -javaagent:/Users/gaborsomogyi/trace-agent/target/trace-agent-1.0-SNAPSHOT.jar -jar target/hadoop-token-trace-1.0-SNAPSHOT-jar
...
TraceAgent (trace_args_with_method_call): public default org.apache.hadoop.security.token.Token[] org.apache.hadoop.security.token.DelegationTokenIssuer.addDelegationTokens(java.lang.String,org.apache.hadoop.security.Credentials) throws java.io.IOException parameter instance with index 1 method call "getAllTokens" returns with 
[Kind: testKind, Service: testService, Ident: 74 65 73 74 49 64 65 6e 74 69 66 69 65 72]
...
```

### CODE REVIEW

**Comment by gaborgsomogyi** [comment]
cc @attilapiros

File: src/main/java/net/test/TraceAction.java [hunk 1]
```diff
@@ -7,8 +7,6 @@
 
 import static net.bytebuddy.matcher.ElementMatchers.*;
 
-import java.util.Map;
```
Comment (gaborgsomogyi) [inline src/main/java/net/test/TraceAction.java:10]: Unrelated fix.

File: README.md [hunk 2]
```diff
@@ -296,6 +296,28 @@ KafkaClient {
 	useKeyTab=true
 	useTicketCache=false
 	serviceName=kafka
+};
```
Comment (gaborgsomogyi) [inline README.md:299]: Unrelated fix.

File: src/main/java/net/test/interceptor/TraceParamCallRetValueInterceptor.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  }
+
+  @RuntimeType
+  public Object intercept(
+      @Origin Method method, @AllArguments Object[] allArguments, @SuperCall Callable<?> callable)
+      throws Exception {
+    String retVal = "";
+    if (allArguments.length > paramIndex) {
+      try {
+        Method getAllTokens = allArguments[paramIndex].getClass().getMethod(methodToCall);
```
Comment (attilapiros) [inline src/main/java/net/test/interceptor/TraceParamCallRetValueInterceptor.java:48]: `getAllTokens` -> `methodToCall`

File: README.md [hunk 4]
```diff
@@ -296,6 +296,28 @@ KafkaClient {
...
+```
+
+# trace_param_call_retval action
```
Comment (attilapiros) [inline README.md:302]: better: `trace_args_with_method_call`

File: src/main/java/net/test/interceptor/TraceParamCallRetValueInterceptor.java (hunk 3 above)
Comment (gaborgsomogyi) [reply src/main/java/net/test/interceptor/TraceParamCallRetValueInterceptor.java:48]: Nice catch, fixed.

File: README.md (hunk 4 above)
Comment (gaborgsomogyi) [reply README.md:302]: Renamed.

**Comment by attilapiros** [comment]
Thanks for the contribution!
LGTM
